src/pages/docs/data-folder.mdx

```diff
@@ -0,0 +1,129 @@
+Logs from the application are stored here. This is useful for troubleshooting and monitoring the application's behavior over time.
+
+### `models/`
+Stores the AI models that the assistants use to process requests and generate responses.
+
+### `settings/`
+General settings for the application are stored here, separate from individual assistant or engine configurations.
+
+### `threads/`
+Threads history is kept in this directory. Each session or thread is stored in a way that makes it easy to review past interactions.
+
+### `extensions/`
+Extensions enhance Jan's functionality by adding new capabilities or integrating external services.
+
+## Detailed File Layout
+Each of the primary folder structures should have the following detailed structure:
+```yaml
+/assistants
+  /jan                                  
+    assistant.json                      
+  /shakespeare                          
+    assistant.json
+/engines                
+  engine_A.json                         
+/extensions
+  extensions.json                       
+  /@janhq               
+    /extension_A
+     package.json                       
+/logs
+  /app.txt
+/models
+  /model_A
+    model.json                          
+/settings
+  settings.json                         
+  /@janhq               
+    /extension_A_Settings
+     settings.json 
+/threads
+  /jan_thread_A
+   messages.jsonl                       
+   thread.json                          
+  messages.jsonl                        
+```
+
+### Assistants
+- **Default Assistant**: Located in `/assistants/jan/`, it includes an `assistant.json` configuring the default settings and capabilities.
+- **Custom Assistant Example**: The `/assistants/shakespeare/` shows a custom setup, also with its own `assistant.json`.
+
+### Engines
+- **Engine Configurations**: Files like `engine_A.json` in the `/engines/` folder configure how Jan can interact with the respective inference engines.
+
+### Extensions
+- **Extension Configuration**: The `extensions.json` in the `/extensions/` directory provides settings for all installed extensions.
+- **Specific Extensions**: Subdirectories like `/@janhq/extension_A/` contain `package.json` files which import necessary modules for each extension.
+
+### Logs
+- **Application Logs**: The file `/logs/app.txt` captures general application activity.
+
+### Models
+- **Model Configurations**: Each model directory, such as `/models/modelA/`, contains a `model.json` with settings specific to that model.
+
+### Settings
+- **General Settings**: The `settings.json` in the `/settings/` directory holds application-wide settings.
+- **Extension-specific Settings**: Additional settings for extensions are stored in respective subdirectories under `/settings/@janhq/`.
+
+### Threads
+- **Threads History**: Each thread is stored in its subdirectory, such as `/threads/jan_unixstamp/`, with files like `messages.jsonl` and `thread.json` detailing the thread settings.
+
+## Verify Installation
```

Comment:
This section should be renamed: "Open the Data Folder"

To open the Data Folder directly from the Jan application, follow...